ui-components-old/src/main/java/com/ivy/old/theme/modal/edit/AmountModal.kt

```diff
@@ -454,23 +457,25 @@ fun KeypadCircleButton(
     )
 }
 
-@SuppressLint("ComposableModifierFactory", "ModifierFactoryExtensionFunction")
+@SuppressLint("ComposableModifierFactory", "ModifierFactoryExtensionFunction", "MissingPermission")
```

Comment:
Would the app crash if there's no permission?

Reply:
Ideally, it shouldn't crash because i also tested it without any permission